sdk/cargo-build-sbf/src/main.rs

```diff
@@ -968,6 +1022,24 @@ fn main() {
     let sbf_sdk: PathBuf = matches.value_of_t_or_exit("sbf_sdk");
     let sbf_out_dir: Option<PathBuf> = matches.value_of_t("sbf_out_dir").ok();
 
+    let sbf_tools_version = if matches.is_present("tools_version") {
```

Comment:
The unwraps could be avoided with something like this:
```suggestion
    let sbf_tools_version = if let Ok(tools_version) = matches.value_of_t("tools_version") {
```